src/androidTest/java/com/mixpanel/android/mpmetrics/MixpanelNotificationBuilderTest.java

```diff
@@ -108,6 +119,32 @@ public void testInvalidIcon() {
         verify(builderSpy).setSmallIcon(DEFAULT_ICON_ID);
     }
 
+    public void testExpandedImageUsingValidUrl() throws ImageStore.CantGetImageException{
+        final Intent intent = new Intent();
+        intent.putExtra("mp_message", "MESSAGE");
+        intent.putExtra("mp_imgurl", VALID_URL);
+
+        Bitmap fakeBitmap = getFakeBitmap();
+
+        when(mpPushSpy.getBitmapFromUrl(VALID_URL)).thenReturn(fakeBitmap);
+
+        mpPushSpy.createNotification(intent);
+        verify(mpPushSpy).getBitmapFromUrl(VALID_URL);
+        verify(builderSpy).setStyle(new Notification.BigPictureStyle().bigPicture(fakeBitmap));
+    }
+
+    public void testExpandedImageUsingInvalidUrl() throws ImageStore.CantGetImageException{
+        final Intent intent = new Intent();
+        intent.putExtra("mp_message", "MESSAGE");
+        intent.putExtra("mp_imgurl", "");
+
+        when(mpPushSpy.getBitmapFromUrl(VALID_URL)).thenReturn(null);
+
+        mpPushSpy.createNotification(intent);
+        verify(mpPushSpy).getBitmapFromUrl(VALID_URL);
```

Comment:
the only reason this passes is because it gets called on line 141. I'd remove that line and modify this to verify `getBitmapFromUrl` is never called.

Reply:
Changed these tests they should actually verify useful calls in next PR